src/robot/reporting/xunitwriter.py

```diff
@@ -57,6 +60,14 @@ def _get_stats(self, statistics):
         )
 
     def end_suite(self, suite):
+        if suite.metadata or suite.doc:
+            self._writer.start('properties')
+            if suite.doc:
+                self._writer.element('property', attrs={'name': 'Suite Documentation', 'value': suite.doc})
```

Comment:
Just `Documentation` would probably be better than `Suite Documentation`. We use that in the data as well.